src/SerializeResult.cc

```diff
@@ -150,9 +153,10 @@ sos::Object WrapParseResultRefract(snowcrash::ParseResult<snowcrash::Blueprint>&
 }
 
 sos::Object drafter::WrapResult(snowcrash::ParseResult<snowcrash::Blueprint>& blueprint,
-                                const WrapperOptions& options)
+                                const WrapperOptions& options,
+                                ConversionContext& context)
```

Comment:
In curent time it seems, but we will need merge, but we will need to merge Errors/Warnings at least there https://github.com/apiaryio/drafter/blob/master/src/main.cc#L82